src/AzureIoTHubClient.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Arduino. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <time.h>
+#include <sys/time.h>
+
+#include <AzureIoTHubClient.h>
+
+Client* AzureIoTHubClient::sslClient = NULL;
+
+AzureIoTHubClient::AzureIoTHubClient(Client& sslClient)
+{
+    AzureIoTHubClient::sslClient = &sslClient;
+}
+
+void AzureIoTHubClient::begin()
+{
+#if defined(ARDUINO_ARCH_ESP8266)
+    Serial.setDebugOutput(true);
```

Comment:
> The setDebugOutput call would be needed to see wifi diagnostic output also (outside the scope of the azure library) so feels natural to have it with serial setup in the example

Ok, I'll move it to the sketch then.

Could you clarify if it needs to be set to true for the library to function like the SAMD side? Also, does this mean WiFi diagnostic output will be enabled?